src/main/java/com/google/api/generator/engine/format/Comment.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.api.generator.engine.format;
+
+public interface Comment {
+  // format the comment
+  public String write();
```

Comment:
Same as the Visitor pattern comment above, this class doesn't need a `write()` method. Instead, this class should have a "comment()" class that returns the final, consolidated comment.

Reply:
sure, now it only has only `comment()` method to gather comments.